application/src/main/java/org/thingsboard/server/service/install/InstallScripts.java

```diff
@@ -260,4 +266,45 @@ public void createOAuth2Templates() throws Exception {
             );
         }
     }
+
+    public void loadSystemLwm2mResources() {
+        Path resourceLwm2mPath = Paths.get(dataDir, MODELS_LWM2M_DIR);
+        try (DirectoryStream<Path> dirStream = Files.newDirectoryStream(resourceLwm2mPath, path -> path.toString().endsWith(InstallScripts.XML_EXT))) {
+            dirStream.forEach(
+                    path -> {
+                        try {
+                            String data = Base64.getEncoder().encodeToString(Files.readAllBytes(path));
+                            TbResource tbResource = new TbResource();
+                            tbResource.setTenantId(TenantId.SYS_TENANT_ID);
+                            tbResource.setData(data);
+                            tbResource.setResourceType(ResourceType.LWM2M_MODEL);
+                            tbResource.setFileName(path.toFile().getName());
+                            doSaveLwm2mResource(tbResource);
+                        } catch (Exception e) {
+                            log.error("Unable to load resource lwm2m object model from file: [{}]", path.toString());
+                            throw new RuntimeException("resource lwm2m object model from file", e);
+                        }
+                    }
+            );
+        } catch (Exception e) {
+            log.error("Unable to load resources lwm2m object model from file: [{}]", resourceLwm2mPath.toString());
+            throw new RuntimeException("resource lwm2m object model from file", e);
+        }
+    }
+
+    private void doSaveLwm2mResource(TbResource resource) throws ThingsboardException {
+        try {
+            log.trace("Executing saveResource [{}]", resource);
+            if (StringUtils.isEmpty(resource.getData())) {
+                throw new DataValidationException("Resource data should be specified!");
+            }
+            toLwm2mResource(resource);
+            resourceService.saveResource(resource);
+        } catch (DataValidationException e) {
+            log.error("[{}] {}", resource.getFileName(), e.getMessage());
```

Comment:
Please use debug here. We don't need these errors in the install log.

Reply:
Ok